src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 101;
```

Comment:
can't see reason for naming used
```suggestion
    private static final int BALL_NUMBER_LIMIT = 101;
```